### DISCUSSION TITLE
Cannot figure out exclusive solo

### DISCUSSION THREAD

**Discussion by grnprplorngyllw** [helgoboss/helgobox#368] [Q&A]
Hello! I need help, please.

I have a drumset on 10 tracks and 1 reference track with a song. I soloed snare. Good. Now, I want to listen ONLY to my reference track. Good, I can do it thanks to exclusive solo. Now I want to go back to snare solo. I unsolo the reference track and my problem: all the drumset tracks get soloed, when I only need to solo the snare track.

I really dont ever need to listen to my reference track together with other tracks. NEVER. But sometimes I am working on a soloed guitar, and I want to easily jump back and forth a reference track and the soloed guitar. I appreciate any tips.

I tried press only, but I dont know. it doesnt do what i need. Please help!

**Comment by helgoboss** [comment]
Answered in forum already.
